authorization/role/service/role_management_service.go

```diff
@@ -95,3 +109,29 @@ func (r *RoleManagementServiceImpl) ListByResourceAndRoleName(ctx context.Contex
 
 	return roles, err
 }
+
+// Assign assigns a user/identity to role, for a specific resource
+func (r *RoleManagementServiceImpl) Assign(ctx context.Context, identityIDs []uuid.UUID, resourceID string, roleName string) error {
+
+	// TODO: Check if the user is authorized to assign roles to other users.
+	var err error
+	return application.Transactional(r.db, func(appl application.Application) error {
+		err = appl.ResourceRepository().CheckExists(ctx, resourceID)
```

Comment:
Another unnecessary DB call IMO.
So, there are four DB calls total instead of just one.

Reply:
Could you check again, please ? 
- after the refactoring.